### ISSUE TITLE
Can not connect to VNC

### ISSUE THREAD

**Issue by mareklibra** [kubevirt/kubevirt#1893]
/kind bug

**What happened**: VNC session cannot be retrieved repeatadly

**What you expected to happen**: I can repetadly connect-disconnect to VNC

**How to reproduce it (as minimally and precisely as possible)**:
- have a running VM (tested with vm-fedora.yml)
- virtctl vnc vm-fedora
- close the remote-viewer window or ctrl-c
- repeat previous 2 steps a few times (in my case 4 is enough)
- suddenly, the remote-viewer window does not show up as the session via API can not be established

**Anything else we need to know?**:
Originally observed with noVNC, but is reproducible with virtctl as well.
Error reported by the RFB: `Invalid server version ual Mac`

Can not find relevant additional info in virt-api or vmi logs.

**Environment**:
- KubeVirt version (use `virtctl version`): v0.11.0
- Kubernetes version (use `kubectl version`): v1.11.0+d4cacc0 , OC: v3.11.51
- VM or VMI specifications:
- Cloud provider or hardware configuration: Upshift (PNQ2 cluster)
- OS (e.g. from /etc/os-release): RHEL 7.6
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

**Comment by mareklibra** [comment]
vm-fedora.yml:
```
apiVersion: kubevirt.io/v1alpha2
kind: VirtualMachine
metadata:
  labels:
    kubevirt.io/vm: vm-fedora
  name: vm-fedora2
spec:
  running: false
  template:
    metadata:
      labels:
        kubevirt.io/os: fedora28
        kubevirt.io/vm: vm-fedora
    spec:
      domain:
        devices:
          disks:
            - disk:
                bus: virtio
              name: containerdisk
              volumeName: registryvolume
            - disk:
                bus: virtio
              name: cloudinitdisk
              volumeName: cloudinitvolume
        machine:
          type: ''
        resources:
          requests:
            memory: 256M
      terminationGracePeriodSeconds: 0
      volumes:
        - name: registryvolume
          containerDisk:
            image: 'kubevirt/fedora-cloud-registry-disk-demo:latest'
        - cloudInitNoCloud:
            userData: |-
              #cloud-config
              password: fedora
              chpasswd: { expire: False }
          name: cloudinitvolume
```

**Comment by fabiand** [comment]
Is this still an issue @mareklibra ?

**Comment by lxs137** [comment]
I meet this problem in v0.14.0 too. If I connect-disconnect vnc client serval times, this bug can be reproduced definitely.

**Comment by rmohr** [comment]
Yes. Reproduced it. Logs of `virt-api` say

```
{"component":"virt-api","kind":"VirtualMachineInstance","level":"error","msg":"error ecountered reading from websocket stream","name":"vmi-replicaset-cirros95jh4","namespace":"default","pos":"subresource.go:127","reason":"read tcp 10.244.0.34:8443-\u003e10.244.0.1:45830: use of closed network connection","timestamp":"2019-03-01T12:59:51.018397Z","uid":"b99909ab-3c21-11e9-a097-525500d15501"}
```

**Comment by lxs137** [comment]
Probably because of the complex way to connect vnc (`virtctl -> kubectl -> apiserver -> vnc.sock`)? 

Will you release any convenient vnc-connect tools?

**Comment by rmohr** [comment]
Fixed in #2171
